apps/crn-frontend/src/network/teams/TeamProfile.tsx

```diff
@@ -74,6 +74,8 @@ const TeamProfile: FC<TeamProfileProps> = ({ currentTime }) => {
   );
 
   if (team) {
+    const sharedOutputsCount = team.outputs?.length || 0;
```

Comment:
Could we default this inside the component? It be cleaner

Reply:
Sure, I thought you would prefer this way to keep it like the other two counts, where there's no need to use a default value, but changing it